### ISSUE TITLE
Two ideas, at least one I think you have mentioned on Facebook

### ISSUE THREAD

**Issue by jonathancolledge** [cagnulein/qdomyos-zwift#824]
Idea 1: integrate with a web app such as this https://github.com/stefaandesmet2003/hometrainer to play videos using a GPX file to control the speed of the video and trainer resistance (if I understand correctly, you have most of this done as you can read GPX). Maybe all it needs is the above to support DIRCON!? Or just a modification of the Bluetooth support? There is a demo here: https://stefaandesmet2003.github.io/hometrainer/index.html but it only supports one trainer, I think.

I have a few cycling videos that I have made. I am working very slowly as I have little time, but I'd like to show you some examples in the next two weeks.

Idea 2: I think you mentioned on Facebook about following Google maps satellite view - is that correct or am I imagining? Anyway, a chap called Andreas Otten has done this with ANT+ and Unity. I am yet to upload my preview to YouTube (I think I should redo it). I don't know what could be done, but maybe Bluetooth or DIRCON could be used. I don't know if he would collaborate, as he is also hard at work on FreeGroupRide. That is ANT only. If you imagine a way of bringing Bluetooth to the table it, might be worth a discussion. Unless you are already some way down this route, in which case I look forward to seeing it! Another example of the map with a different view is https://github.com/floschnell/trainerjs

Idea 3: I have just thought of! As above, offering Bluetooth support to FreeGroupRide (Unity based) https://freegroupride.de/index.html Facebook link at bottom of page.

I only mention these things as ideas. I don't need them, but just thought I should highlight them. I like the idea of a kind of supplemental browser-based app that QZ users could start on any platform, and it would either show a graph of the workout, or a video (maybe workout overlay), Ideally no hosting, but just something that can be downloaded and loaded in the browser and work offline. A bit like the first example, but QZ branded. 

I won't take offence if you aren't interested in any of this though!

**Comment by cagnulein** [comment]
about the first one i think @p3g4asus already did it, isn't it? maybe @p3g4asus can show its actually setup

**Comment by cagnulein** [comment]
yes for the second one it's already developed and it's what i'm using for my daily training. you can test it on android or ios loading a gpx file. i would like to create a video about this in order to show how you can use this. @janabouc asks me the same :)

**Comment by cagnulein** [comment]
3) i didn't know this project. I have to dig in more. I will let you know! Thanks for sharing all your ideas!

**Comment by cagnulein** [comment]
OMG Cesium looks so good! https://cesium.com/ I didn't know about this!

**Comment by cagnulein** [comment]
@jonathancolledge https://www.youtube.com/watch?v=V4QlQTmGbYY

**Comment by janabouc** [comment]
Wow! Great video. I will add this to the guide! I was amazed to see the map appear inside QZ. It won’t do street view though I guess? But topographical is cool! 

To make it worth using you need to have a controllable treadmill, bike or SS2K, right? Unless you’re happy to just be following along on a topo map. I’m guessing it shows more details in the city?
